gateways/adyen/request_builder.go

```diff
@@ -103,6 +103,10 @@ func buildLevel3Data(level3Data *sleet.Level3Data) map[string]string {
 			break
 		}
 		keyBase = fmt.Sprintf("enhancedSchemeData.itemDetailLine%d.", idx+1)
+		// Due to issues with the credit card networks, dont send any line item if discount amount is 0
+		if lineItem.ItemDiscountAmount.Amount > 0 {
+			additionalData[keyBase+"discountAmount"] = sleet.AmountToString(&lineItem.ItemDiscountAmount)
```

Comment:
nit should we add whitespace around `+`?

Reply:
Like idx + 1? Sure